src/components/common/TranslationWrapper.tsx

```diff
@@ -14,6 +14,8 @@ const TranslationWrapper = ({ children }: Props): JSX.Element => {
   const { hooks } = useContext(QueryClientContext);
   const { data: member } = hooks.useCurrentMember();
 
+  const [i18n] = useState(i18nConfig);
```

Comment:
?
```suggestion
  const [i18n] = useState(i18nConfig());
```

Reply:
Actually here I use the functional form of the useState to create the instance for the lifetime of the component. it is similar to:
```ts
const. [i18n] = useState(() => i18nConfig());
```